docker/base/Dockerfile

```diff
@@ -24,6 +24,14 @@
 ARG bootstrap_version=9
 ARG image="vitess/bootstrap:${bootstrap_version}-mysql57"
 
+FROM node:16-bullseye-slim as node
+
+COPY web/vtadmin /vt/web/vtadmin
+
+# install/build/clean web dependencies
+RUN npm --prefix /vt/web/vtadmin ci && \
+    npm run --prefix /vt/web/vtadmin build
+
```

Comment:
We should do the build of the vtadmin binaries in the k8s vtadmin docker file itself. We only use this build for that image, so it doesn't make sense to add it to all the other images.